spec/permissions/decidim/initiatives/permissions_spec.rb

```diff
@@ -212,24 +212,24 @@
           .and_return(true)
       end
 
-      it { is_expected.to be false }
+      it { is_expected.to be true }
 
-      context "when authorizations are not required" do
+      context "when authorizations are required" do
```

Comment:
Do we enforce authorization on initiative creation  ?

Reply:
No we don't, it's the default behaviour to not require authorization